src/opera/pge/rtc_s1/rtc_s1_pge.py

```diff
@@ -249,6 +260,51 @@ def _rtc_filename(self, inter_filename):
 
         return rtc_filename
 
+    def _static_layer_filename(self, inter_filename):
+        """
+        Returns the file name to use for the static layer RTC products produced
```

Comment:
I'm being a nit-picker, but maybe:  Returns the file name of the static layer.....